controlpanel/api/cluster.py

```diff
@@ -63,6 +73,17 @@ def create(self):
             f"--set=Username={self.user.slug}",
         )
 
+    def reset_home(self):
+        """
+        Reset the user's home directory.
+        """
+        helm.upgrade_release(
+            f"reset-user-home-{self.user.slug}",
+            f"{settings.HELM_REPO}/reset-user-home",
+            f"--namespace=user-{self.user.slug}",
```

Comment:
Not that it makes any difference as it yields the same result but as you can see from `Helm.upgrade_release()` call above you may want to use the `self.k8s_namespace` which returns the user namespace.

As I said, it's the same string at the end of the day but I thought you may want to know that there is a method/property in the user model for that.